lib/elastomer/client/delete_by_query.rb

```diff
@@ -0,0 +1,82 @@
+module Elastomer
+  class Client
+
+    class DeleteByQuery
+
+      def initialize( client, query, params = nil )
+        @client    = client
+        @query     = query
+        @params    = params
+        @responses = []
+      end
+
+      attr_reader :client, :query, :params, :responses
+
+      def is_ok( status )
```

Comment:
Normally methods that return a boolean value are suffixed with a `?`, e.g. `is_ok?`